templates/sfm/dashboard.html

```diff
@@ -5,32 +5,33 @@
     <div class="col-sm-12">
         <div class="jumbotron">
             <form action='{% url 'search' %}' method='GET'>
-                <p>Investigate <strong>police</strong> and <strong>military commanders</strong>, <strong>units</strong> and <strong>areas of operation</strong></p>
+                {% comment %}Translators: Please keep the "strong" tags, which display the numbers in bold{% endcomment %}
```

Comment:
This is cool! Were you able to confirm that it works?

Reply:
Yes, it does!